products/statement-generator/src/components/FormHeader.tsx

```diff
@@ -127,6 +131,12 @@ const FormHeader = () => {
     formState: { involvement },
   } = useContext(FormStateContext);
 
+  const location = useLocation();
+  const isFormRoute =
+    location.pathname.startsWith('/form/') || location.pathname === '/form';
+
+  const headerClass = isFormRoute ? '' : classes.hidden;
+
```

Comment:
Rather than handling this with a custom 'hidden' class, we should use the location to return early if `isFormRoute` is true.